src/fprime/cookiecutter_templates/cookiecutter-fprime-project/cookiecutter.json

```diff
@@ -1,6 +1,6 @@
 {
     "project_name": "MyProject",
-    "fprime_branch_or_tag": "master",
+    "fprime_branch_or_tag": "devel",
```

Comment:
Why?  I thought we wanted master.

Reply:
I noted switching back to master as Future Work for next release

Reply:
Oh wait - I thought the change I made was from devel to master. Lol. Will revert back.